packages/augur-simplified/src/modules/market/trading-form.tsx

```diff
@@ -305,26 +311,27 @@ const TradingForm = ({
   const [breakdown, setBreakdown] = useState(getEnterBreakdown(null));
   const [amount, setAmount] = useState<string>("");
   const [tradeEstimates, setTradeEstimates] = useState<TradeEstimates>({})
+  const [tradeError, setTradeError] = useState<string>(null);
   const ammCash = amm?.cash;
   const outcomes = amm?.ammOutcomes || [];
   const userCashBalance = amm?.cash?.name ? balances[amm?.cash?.name]?.balance : "0";
 
   useEffect(() => {
     let isMounted = true;
     const getEstimate = async () => {
+      setTradeError(null);
```

Comment:
if you move this line, the button won't flash. can you only put it after you know there are no errors?